test.sh

```diff
@@ -29,6 +29,9 @@ PKG=$(go list -f '{{if .TestGoFiles}} {{.ImportPath}} {{end}}' ./...)
 
 go test $PKG
 
+# Browser tests are in a separate module, so do not show in $PKG.
+(cd browsertests && go test)
```

Comment:
Will we lose code coverage and race detection through browser tests with this change?

Plus `go vet` checks and formatting. And staticcheck above.

Reply:
Good point. Added staticcheck, vet, race, lint, fmt.

Did not add coverage since browsertests only tests javascript code for which we cannot currently get any coverage info.